fabric-events-interaction-v0/src/testmod/java/net/fabricmc/fabric/test/event/interaction/PlayerBreakBlockTests.java

```diff
@@ -29,16 +29,10 @@ public class PlayerBreakBlockTests implements ModInitializer {
 
 	@Override
 	public void onInitialize() {
-		PlayerBlockBreakEvents.BEFORE.register(((world, player, pos, state, entity) -> {
-			return state.getBlock() != Blocks.BEDROCK;
-		}));
+		PlayerBlockBreakEvents.BEFORE.register(((world, player, pos, state, entity) -> state.getBlock() != Blocks.BEDROCK));
 
-		PlayerBlockBreakEvents.CANCELED.register(((world, player, pos, state, entity) -> {
-			LOGGER.info("Block break event canceled at " + pos.getX() + ", " + pos.getY() + ", " + pos.getZ());
-		}));
+		PlayerBlockBreakEvents.CANCELED.register(((world, player, pos, state, entity) -> LOGGER.info("Block break event canceled at {}, {}, {} (client-side = {})", pos.getX(), pos.getY(), pos.getZ(), world.isClient())));
 
-		PlayerBlockBreakEvents.AFTER.register(((world, player, pos, state, entity) -> {
-			LOGGER.info("Block broken at " + pos.getX() + ", " + pos.getY() + ", " + pos.getZ());
-		}));
+		PlayerBlockBreakEvents.AFTER.register(((world, player, pos, state, entity) -> LOGGER.info("Block broken at {}, {}, {} (client-side = {})", pos.getX(), pos.getY(), pos.getZ(), world.isClient())));
```

Comment:
No need to change this.

Reply:
Should I not add a "client=false" to the print so the side can be identified?

Reply:
Oh,I see I missed that change, this is fine then. Sorry